.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Run tests
+        run: make test MVN_FLAGS=-B -e
+      - name: Set version
+        run: ./mvnw versions:set -DgenerateBackupPoms=true -DnewVersion=${{ github.ref_name }} --no-transfer-progress
+      - name: Build
+        run: make build MVN_FLAGS=-B -e
+      - name: Test
+        run: make test MVN_FLAGS=-B -e
+      - name: Release Core
```

Comment:
Why release core and cli separately? One `mvn deploy` would upload all the modules in a single action.

Reply:
I did not know - not very maven battle proven on those grounds. i use gradle in basically all projects. I'll fix that, good catch

Reply:
running mvn deploy on the parent project know, hopefully this is what does the expected. Thanks for the hint

Reply:
Thanks.

With the move to my package manager I'll need to make some updates to my environment to enable temporary credentials on GH, so I'll take over this PR to finalise it if that is ok.